README.md

```diff
@@ -190,29 +187,29 @@ config :my_app, MyApp.Mailer,
 ```
 
 You can create new adapters for any environment by implementing the
-[`Bamboo.Adapter`] behaviour.
+[`Bamboo.Adapter`] behavior.
 
 ## Delivering Emails in the Background
 
-Often times you don't want to send email right away because it can block
+Often times you don't want to send an email right away because it can block
 process completion (e.g. a web request in Phoenix). Bamboo provides a
 `deliver_later` function on your mailers to send emails in the background. It
-also provides a [`Bamboo.DeliverLaterStrategy`] behaviour that you can
+also provides a [`Bamboo.DeliverLaterStrategy`] behavior that you can
 implement to tailor your background email sending.
 
 By default, `deliver_later`uses [`Bamboo.TaskSupervisorStrategy`]. This
-strategy sends the email right away, but does so in the background without
+strategy sends the email right away but does so in the background without
 linking to the calling process, so errors in the mailer won't bring down your
 app.
 
 You can also create custom strategies by implementing the
-[`Bamboo.DeliverLaterStrategy`] behaviour. For example, you could create
+[`Bamboo.DeliverLaterStrategy`] behavior. For example, you could create
```

Comment:
The previous spelling was the correct one. Behaviours (with the `u`) are part for [Elixir and Erlang](https://hexdocs.pm/elixir/1.3.4/behaviours.html). So we want to keep the other spelling to communicate that we mean an Elixir/Erlang behaviour.